src/dataSource/data.json

```diff
@@ -0,0 +1,61 @@
+[
+  {
+    "id": 1,
+    "symptom": "For a few days I've had sharp stomach pain and sickness alongside no appetite at all. Then last night the pain grew even more intense, I vomited for at least three hours and felt sick all through the night.",
+    "symptomName": "Appendicitis",
+    "terms": [
+      "high temperature",
+      "feeling hot",
+      "shivery",
+      "headaches",
+      "muscle and joint pain",
+      "feel tired",
+      "exhausted",
+      "complete loss of energy",
+      "heart palpitations"
+    ],
+    "reportIds": [1]
```

Comment:
This is 1 where the rest are empty? is this the source? social or academia? did not follow the data topics enough